src/quo2/components/tag.cljs

```diff
@@ -0,0 +1,45 @@
+(ns quo2.components.tag
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo.theme :as theme]
+            [quo2.components.text :as text]))
+
+(def themes {:light {:background-color colors/neutral-20}
+             :dark {:background-color colors/neutral-80}})
+
+(defn get-value-from-size [size big-option small-option]
+  (if  (= size :big) big-option small-option))
+
+(defn tag-container [size]
+  {:height (get-value-from-size size 32 24)
+   :align-items :center
+   :flex-direction :row
+   :border-radius 20})
+
+(defn tag  "[tag opts]
```

Comment:
No strong opinion, but it will be great if text-value/label is passed as argument, so that all components follow same pattern
Ex. [1](https://github.com/status-im/status-mobile/blob/8e3258769e7b1e6b059d403ffe5337871635bea5/src/quo2/components/button.cljs#L109) [2](https://github.com/status-im/status-mobile/blob/8e3258769e7b1e6b059d403ffe5337871635bea5/src/quo2/components/counter.cljs#L22)

Reply:
thanks Parvesh, I'll adjust 👍